app/routes/router.js

```diff
@@ -92,6 +92,16 @@ router
     .get('/findalltransactionsbyparamid', function (req, res){
         API.findTransactionsByParamId(req, res)
     })
+	
+	// gets all the errors
+    .get('/findallerrorreports', function (req, res){
```

Comment:
@seancavanaugh ok, this is fine for now as long as we get the endpoints and docs updated soon.